subsys/emul/espi/emul_espi_host.c

```diff
@@ -217,6 +217,19 @@ int emul_espi_host_port80_write(const struct device *espi_dev, uint32_t data)
 	return 0;
 }
 
+#ifdef CONFIG_ESPI_PERIPHERAL_ACPI_SHM_REGION
+int emul_espi_host_set_acpi_shm(const struct device *espi_dev, uint8_t *data,
+				size_t size)
+{
+	struct emul_espi_driver_api *api =
```

Comment:
Here we would set the mmap memory stored in the host data.

Reply:
Done.